fun.sql

```diff
@@ -1,3 +1,3 @@
 SELECT point_differential
 FROM nba_games
-WHERE nba_team = 'Heat!';
+WHERE nba_team = 'Bulls!';
```

Comment:
Heema, great work but the Warriors are actually the best team. Please change before I approve.